packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -179,6 +180,12 @@ class RenderEditable extends RenderBox {
       ..onTap = _handleTap;
     _longPress = LongPressGestureRecognizer(debugOwner: this)
       ..onLongPress = _handleLongPress;
+    // If the user has not specified a custom cursor, we will render a native
```

Comment:
The 'user' is the material/cupertino layers which are still us :) and those layers should specify